docs/libopenblas.md

```diff
@@ -0,0 +1,34 @@
+# libopenblas
+
+libopenblas is a fairly low-level library, and can get pulled in transitively
+via dependencies. e.g., [tgen](https://github.com/shadow/tgen) uses libigraph,
+which links against liblapack, which links against blas.
+
+## Deadlocks due to `sched_yield` loops
+
+libopenblas, when compiled with pthread support, makes extensive use of
+spin-waiting in `sched_yield`-loops, which currently result in deadlock under
+Shadow.
+
+There are several known workarounds:
+
+* Use a different implementation of libblas. e.g. on Ubuntu, there are several
+  alternative packages that can [provide
+  libblas](https://packages.ubuntu.com/hirsute/libblas.so.3).  In particular,
```

Comment:
Does the linebreak here break the hyperlink in the rendered output?

Reply:
Nope.

PS ... -> View file to see rendered output)
PPS Is there a name for a ... menu? Pacman menu...?

Reply:
I don't think I've seen it called a Pacman menu, but I like it! :)